src/extendable_pydantic/_patch.py

```diff
@@ -16,6 +16,8 @@
     from typing_extensions import Annotated
 
 import wrapt
+import pkg_resources
+from packaging import version
```

Comment:
If we do this, `packaging` should be added as a dependency of this library, but...